samples/highcharts/blog/map-china-world-heritage/demo.js

```diff
@@ -0,0 +1,456 @@
+(async () => {
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+    const geojson = Highcharts.topo2geo(topology);
+
+    // Find the geometry for one specific feature
+    const countryGeometry = geojson.features.find(
+        f => f.properties.name === 'China'
+    ).geometry;
+
+    // Create the chart
+    Highcharts.mapChart('container', {
+        chart: {
+            margin: 0
+        },
+
+        title: {
+            text: ''
+        },
+
+        subtitle: {
+            text: ''
+        },
+
+        navigation: {
+            buttonOptions: {
+                align: 'left',
+                theme: {
+                    stroke: '#e6e6e6'
+                }
+            }
+        },
+
+        mapNavigation: {
+            enabled: true,
+            buttonOptions: {
+                alignTo: 'spacingBox'
+            }
+        },
+
+        mapView: {
+            fitToGeometry: countryGeometry
+        },
+
+        tooltip: {
+            useHTML: true,
+            headerFormat: '{point.point.name}',
+            pointFormat: '<br/><img src="{point.image}"/>'
+        },
+
+        legend: {
+            enabled: true,
+            title: {
+                text: 'World heritage sites in China'
+            },
+            align: 'left',
+            symbolWidth: 20,
+            symbolHeight: 20,
+            itemStyle: {
+                textOutline: '1 1 1px rgba(255,255,255)'
+            },
```

Comment:
The white text outline is hardly visible on the 0.8 opacity white background. Probably better to remove it to keep the config simpler.